cmd/main.go

```diff
@@ -16,18 +20,30 @@ func main() {
 func email() {
 	from := os.Getenv("EMAIL_SENDER_USER")
 	password := os.Getenv("EMAIL_SENDER_PASSWORD")
-
-	toEmail := defines.To
-	to := []string{toEmail}
-
 	host := os.Getenv("EMAIL_HOST")
 	port := os.Getenv("EMAIL_PORT")
-	address := host + ":" + port
-	subject := defines.TitleEmail
-	body := defines.BodyEmail
-	message := []byte(subject + body)
 	auth := smtp.PlainAuth("", from, password, host)
-	err := smtp.SendMail(address, auth, from, to, message)
+	address := host + ":" + port
+	toEmail := defines.To
+	to := []string{toEmail}
+	t, err := template.ParseFiles("./templates/body.html")
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	buf := new(bytes.Buffer)
+	msg := domain.Body{
+		Name:        "Jonathan",
+		TimeStamp:   time.Now(),
+		Temperature: 30,
+	}
+	err = t.Execute(buf, msg)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	body := buf.String()
```

Comment:
Te falta especificar el [MIME](https://developer.mozilla.org/es/docs/Web/HTTP/Basics_of_HTTP/MIME_types) para que el mensaje lo interprete como HTML.
Tambien te falta agregar el header `Subject: <titulo del email>`

```suggestion
 	body := defines.TitleEmail + "\r\n
  	body += "MIME-version: 1.0;\nContent-Type: text/html; charset=\"UTF-8\";" + "\r\n"
 	body += buf.String()
```